Postgrest/src/commonMain/kotlin/io/github/jan/supabase/postgrest/Postgrest.kt

```diff
@@ -45,6 +32,12 @@ import kotlinx.serialization.json.JsonElement
  */
 sealed interface Postgrest : MainPlugin<Postgrest.Config>, CustomSerializationPlugin {
 
+    /**
+     * Instance of Authenticated Supabace Api to interact with Supabase APIs
+     * @see io.github.jan.supabase.network.SupabaseApi
+     */
+    val api: AuthenticatedSupabaseApi
```

Comment:
Not sure if the api object should be exposed, as no other plugin does it

Reply:
Sure we don't need to expose api object. Thanks for pointing this out Jan